wpimath/src/main/native/include/frc/kinematics/ChassisSpeeds.h

```diff
@@ -133,6 +133,50 @@ struct WPILIB_DLLEXPORT ChassisSpeeds {
                                    fieldRelativeSpeeds.omega, robotAngle);
   }
 
+  /**
+   * Converts a user provided robot-relative set of speeds into a field-relative
+   * ChassisSpeeds object.
+   *
+   * @param vx The component of speed in the x direction relative to the robot.
+   * Positive x is towards the robot's front.
+   * @param vy The component of speed in the y direction relative to the robot.
+   * Positive y is towards the robot's left.
+   * @param omega The angular rate of the robot.
+   * @param robotAngle The angle of the robot as measured by a gyroscope. The
+   * robot's angle is considered to be zero when it is facing directly away from
+   * your alliance station wall. Remember that this should be CCW positive.
+   *
+   * @return ChassisSpeeds object representing the speeds in the field's frame
+   * of reference.
+   */
+  static ChassisSpeeds FromRobotRelativeSpeeds(
+      units::meters_per_second_t vx, units::meters_per_second_t vy,
+      units::radians_per_second_t omega, const Rotation2d& robotAngle) {
+    return {vx * (-robotAngle).Cos() + vy * (-robotAngle).Sin(),
+            -vx * (-robotAngle).Sin() + vy * (-robotAngle).Cos(), omega};
```

Comment:
The robot angle should not be negated.

Reply:
not negating the robot angle would make this the same function as `FromFieldRelativeSpeeds` would it not? or am I missing something

Reply:
Correct, because you're still using the wrong rotation matrix. The only negative sign should be in the top-right of the rotation matrix for this function.

Reply:
I could also do 
```java
return ChassisSpeeds.fromFieldRelative(-robotAngle);
```

Reply:
Since our code is write-few, read-many, we should focus on making the code readable.